specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2019-08-09-preview/vmwarevirtustream.json

```diff
@@ -702,6 +755,40 @@
     }
   },
   "definitions": {
+    "HostQuota": {
+      "description": "Host quotas",
+      "properties": {
+        "gp": {
+          "description": "VS20 host quota",
+          "type": "integer"
+        },
+        "he": {
+          "description": "VS36 host quota",
+          "type": "integer"
+        }
+      }
+    },
+    "QuotaGet": {
+      "description": "Subscription quotas",
+      "properties": {
+        "hostsRemaining": {
+          "$ref": "#/definitions/HostQuota"
+        },
+        "quotaEnabled": {
+          "description": "Subscription has a valid quota",
+          "type": "boolean"
+        }
+      }
+    },
+    "QuotaRegion": {
+      "description": "Quota region request",
+      "properties": {
+        "region": {
```

Comment:
>"region" [](start = 8, length = 8)

I'd suggest using "location" since it is what all resources use to signify their location